packages/gasket-request/README.md

```diff
@@ -0,0 +1,130 @@
+# @gasket/request
+
+The purpose of this package is to provide a consistent request object for Gasket
+plugins and apps to use, regardless of the request handling framework.
+
+## Installation
+
+#### Existing apps
+
+```shell
+npm install @gasket/request
+```
+
+## Functions
+
+### GasketRequest
+
+The `GasketRequest` class is a representation of a request object that can be
+used by Gasket plugins and apps. It is a consistent shape that can be used
+across different request handling frameworks.
+
+A `GasketRequest` object has the following properties:
+- `headers` - (object) Request headers
+- [`cookies`] - (object) Request cookies
+- [`query`] - (object) Query parameters
+- [`path`] - (string) Request path
```

Comment:
Are these square brackets for showing these are optional? If so, when would `path` be missing? I also wonder wonder if it'd be easier if none of these were optional (path would default to `/`, cookies to `{}`, query to `{}`, etc).

Reply:
Yea, good callout. Reminds me I need to add some tests for path, also. It's possible for path NOT to exist for "static" GasketRequest objects for some prerendered page scenarios. Certainly not common. Perhaps we could default to empty string for path, with empty objects for cookies and query.